.github/workflows/pr_infra.yaml

```diff
@@ -10,7 +10,7 @@ on:
       - ready_for_review
     paths:
       # Trigger the workflow when resources are modified
-      - "infra/resources/**"
+      - "infra/resources/prod/westeurope/**"
```

Comment:
Please get rid of line 25 as well

Reply:
yep, we need to remove the `region` parameter (westeurope) which is not used in the new workflow